tremor-codec/src/codec/kafka_schema_registry.rs

```diff
@@ -0,0 +1,180 @@
+// Copyright 2020-2021, The Tremor Team
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+//! The `avro` codec supports Apache Avro binary encoding.
```

Comment:
This should explain something about the codec.

```suggestion
//! The `kafka-schema-registry` codec supports Apache Avro binary encoding.
```